packages/wrangler/bin/wrangler.js

```diff
@@ -76,6 +76,11 @@ Consider using a Node.js version manager such as https://volta.sh/ or https://gi
 			if (process.send) {
 				process.send(message);
 			}
+		})
+		.on("disconnect", () => {
+			if (process.disconnect) {
+				process.disconnect();
+			}
```

Comment:
nit, but any reason this can't be `process.disconnect?.()`?

Reply:
No, but I was mirroring the style of the message event handler above it